src/sentry/static/sentry/app/views/organizationIntegrations/constants.tsx

```diff
@@ -7,3 +7,10 @@ export const colors = {
   [NOT_INSTALLED]: 'gray2',
   [PENDING]: 'yellowOrange',
 };
+
+export const weights = {
+  slack: 10,
+  splunk: 3,
+  webhooks: 100,
+  bitbucket_server: 30,
```

Comment:
Is this done based on some sort of metrics?

Reply:
Also I think a more descriptive name (and maybe a comment on where these values come from) would be helpful.

Reply:
@leeandher these weights are going to be hardcoded. I'll add a comment.